staging/src/k8s.io/api/resource/v1alpha3/types.go

```diff
@@ -785,6 +785,14 @@ type DeviceRequestAllocationResult struct {
 	//
 	// +required
 	Device string `json:"device" protobuf:"bytes,4,name=device"`
+
+	// AdminAccess indicates that administrative access to the device(s)
+	// was requested. A device where the only allocation is for
```

Comment:
Does it make more sense after reading about the feature in the request?

https://github.com/kubernetes/kubernetes/blob/139cc3c659dd1624f7a4bbcc3b07fda79539677a/staging/src/k8s.io/api/resource/v1alpha3/types.go#L454-L462

Reply:
> A device where the only allocation is for administrative access is still available for other allocations.

How is that related to what you shared?

Reply:
The value in the status is a direct copy of the value in the spec.

I suppose that is what I need to call out explicitly. I thought it would be obvious :sweat_smile:

Reply:
> >    A device where the only allocation is for administrative access is still available for other allocations.
> How is that related to what you shared?

That follows from the definition of "admin access": admin access does not block normal usage.

Reply:
I replaced with "AdminAccess is a copy of the AdminAccess value in the request which caused this device to be allocated.".

What that means follows from the definition of that boolean in the request.